pkg/generated/openapi/BUILD

```diff
@@ -11,6 +11,43 @@ openapi_library(
     srcs = ["doc.go"],
     go_prefix = openapi_go_prefix,
     openapi_targets = [
+        "kubernetes/cmd/libs/go2idl/client-gen/test_apis/testgroup/v1",
```

Comment:
Why is this change? Have you run hack/update-all.sh?

Reply:
no, I ran `hack/update-bazel.sh`

Reply:
Could you undo the change? These diffs might be an one-time off.